.github/workflows/ensure-cms-branch.yml

```diff
@@ -0,0 +1,30 @@
+name: Ensure correct CMS branch
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+    paths:
+      - 'src/admin/config.yml' # Skip this action unless the Netlify CMS config file has changed.
```

Comment:
Can we use double quotes when possible for consistency? Thanks.

Reply:
These quotes aren't actually required, so I'll remove them.